stage-0-proposals.md

```diff
@@ -53,3 +59,13 @@ See also the [active proposals](README.md), [stage 1 proposals](stage-1-proposal
 [async-context]: https://github.com/legendecas/proposal-async-context
 [async-context-notes]: https://github.com/tc39/notes/blob/HEAD/meetings/2020-07/july-23.md#async-context-updates--for-stage-1
 [string-trim-characters]: https://github.com/Kingwl/proposal-string-trim-characters
+[array-async-from]: https://github.com/js-choi/proposal-array-async-from
+[explicit-this-param]: https://github.com/gilbert/es-explicit-this
+[explicit-this-param-notes]: https://github.com/tc39/notes/blob/master/meetings/2020-02/february-6.md#syntax-for-explicitly-this-argument-for-stage-1
+[this-arg-reflection]: https://github.com/hax/proposal-function-this
+[this-arg-reflection-notes]: https://github.com/tc39/notes/blob/master/meetings/2020-02/february-6.md#function-thisargumentexpected-property
+[index-from-end]: https://github.com/hax/proposal-index-from-end
+[index-from-end-notes]: https://github.com/tc39/notes/blob/master/meetings/2021-01/jan-28.md#index-from-end-syntax
+[json-slashes-hint]: https://gist.github.com/hax/5691ca8acdf9179e63043857cdc3616b
```

Comment:
this has never been placed on an agenda, and isn't a repo.

Reply:
There are some other proposals in the table also never on the agenda and no repo?

Reply:
The only proposals that should meet that criteria are the ones that predate the new stage process; if there's any in the last 6 years, please let me know.

Reply:
It seems at least URL, `deprecated` and `as` proposals are added after new stage process and never on the agenda?

And if repo is required, I'd like to create a repo and copy the gist to it. :)

Reply:
All of those were in fact presented.

Reply:
Ah? I searched agenda repo and can't find James M Snell 's proposals in any agenda.

Reply:
Hm, you may be right on url and deprecated - i thought at least we’d discussed them, but i can’t find it in the notes. @jasnell?

Reply:
I don't believe they actually came up on the agenda. I'm happy to jump in on the next call to discuss and see if there's interest in moving it forward. I don't have high hopes for `deprecated` moving forward as it's definitely very nebulous, but I would like to have a good conversation around `URL`.

Reply:
In that case, maybe we should move deprecated to inactive and url can go on the next agenda?

Reply:
sounds good to me

Reply:
deprecated moved in 6c6d3ab6c90d43c64008d3edeb8c952d6aed624d; i'll leave it to you to add URL to the agenda :-)